### PR TITLE
[1.12.x] Minor Fixes, See Info

### PR DESCRIPTION
**Pull request by CDAGaming**
## Info
This Commit fixes the items below:
- Deprecation Warnings for Gradle (`>> to doLast, Future-Proofing`)
- Removes Un-needed Characters and Typo Fixes
- Runs genPatches (`1.12.2.json changes, and Item.java.patch`)

### CODE REVIEW

**Comment by liach** [comment]
These "fixes" look largely like decorations. If you want to contribute, pick an issue from this list: https://github.com/MinecraftForge/MinecraftForge/issues?q=is%3Aopen+is%3Aissue+label%3AFeature

**Comment by CDAGaming** [comment]
Still Not Dead yet, stale bot. Gimme a bit to rebase this.

Edit #4664 will Close/Adjust this due to Most of what I put here being similar.

File: README.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ Go to [http://files.minecraftforge.net](http://files.minecraftforge.net)
...
- *Lastest build* there. Latest builds may have newer features  but may be
+ *Latest Build* there. Latest builds may have newer features  but may be
  more unstable as a result. The installer will attempt to install forge
  into your vanilla launcher environment, where you can then create a new
  profile using that version and play the game!
 
 Here is a short video from Rorax showing how to install and setup Forge.
 
-[![HOWTO Install Forge](https://img.youtube.com/vi/lB3ArN_-3Oc/0.jpg)](https://www.youtube.com/watch?v=lB3ArN_-3Oc)
+[![HOW TO Install Forge](https://img.youtube.com/vi/lB3ArN_-3Oc/0.jpg)](https://www.youtube.com/watch?v=lB3ArN_-3Oc)
```
Comment (liach) [inline README.md:14]: Leave it `How To`

**Comment by mezz** [comment]
Closed by this larger PR #4664

**Pull request closed without merging**